cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -67,18 +67,24 @@
  */
 public class KafkaCruiseControlServlet extends HttpServlet {
   private static final Logger LOG = LoggerFactory.getLogger(KafkaCruiseControlServlet.class);
-  private static final Logger ACCESS_LOG = LoggerFactory.getLogger("CruiseControlPublicAccessLogger");
   private final AsyncKafkaCruiseControl _asyncKafkaCruiseControl;
+  private final KafkaCruiseControlConfig _config;
   private final UserTaskManager _userTaskManager;
   private final long _maxBlockMs;
   private final ThreadLocal<Integer> _asyncOperationStep;
   private final Map<EndPoint, Meter> _requestMeter = new HashMap<>();
   private final Map<EndPoint, Timer> _successfulRequestExecutionTimer = new HashMap<>();
+  private final boolean _corsEnabled;
+  private final boolean _enableAccessLog;
 
   public KafkaCruiseControlServlet(AsyncKafkaCruiseControl asynckafkaCruiseControl,
                                    long maxBlockMs,
                                    long sessionExpiryMs,
-                                   MetricRegistry dropwizardMetricRegistry) {
+                                   MetricRegistry dropwizardMetricRegistry,
+                                   KafkaCruiseControlConfig kafkaCruiseControlConfig) {
+    _config = kafkaCruiseControlConfig;
+    _corsEnabled = _config.getBoolean(KafkaCruiseControlConfig.WEBSERVER_HTTP_CORS_ENABLED_CONFIG);
+    _enableAccessLog = _config.getBoolean(KafkaCruiseControlConfig.WEBSERVER_ACCESSLOG_ENABLED);
```

Comment:
`_enableAccessLog` seems to be unused - do we need this private variable in this class?

Reply:
remained post refactor. removing this vairable.